spec/takeaway_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:burger_description) {"Cheeseburger"}
+  let(:burger_price) {4.99}
+  let(:burger_to_s) {"#{burger_description}\t#{burger_price}"}
+  let(:burger) { double :dish, :description => burger_description,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.